src/Mollie/WC/Gateway/Creditcard.php

```diff
@@ -18,6 +18,18 @@ public function __construct()
         $this->hasFieldsIfMollieComponentsIsEnabled();
     }
 
+    public function get_icon() {
+        if ($this->enabledCreditcards()
+            && !is_admin()
+        ) {
+            $output = $this->buildSvgComposed() ?: '';
+        } else{
+            $output = $this->icon ? '<img src="' . WC_HTTPS::force_https_url( $this->icon ) . '" alt="' . esc_attr( $this->get_title() ) . '" />' : '';
+        }
```

Comment:
```suggestion
    public function get_icon() {
         $output = $this->icon ? '<img src="' . WC_HTTPS::force_https_url( $this->icon ) . '" alt="' . esc_attr( $this->get_title() ) . '" />' : '';
         
        if ($this->enabledCreditcards() && !is_admin()) {
            $output = $this->buildSvgComposed() ?: '';
        }
```

You don't need `else` blocks ;)